app/translations/dutch.js

```diff
@@ -0,0 +1,396 @@
+// @flow
+const INPUTS = {
+  inputSelectPlaceholder: 'Selecteer',
+  inputPasswordPlaceholder: 'Wachtwoord',
+  inputEncryptedPlaceholder: 'Beveiligde Sleutel',
+  authPrivateKeyPlaceholder: 'Voer je private key hier in',
+  authWatchPlaceholder: 'Vul hier een NEO adres in',
+}
+
+const MISCELLANEOUS = {
+  'auth.cancel': 'Annuleren',
+  'auth.ledger.connectLedger':
+    'Verbind en ontgrendel je <strong>Ledger-apparaat</strong>',
+  'auth.ledger.navigateToNeoApp':
+    'Navigeer naar de <strong>NEO-app</strong> op je apparaat',
+  'auth.ledger.retry': 'Opnieuw proberen?',
+  'auth.ledger.fetchAddress': 'Haal extra adressen op',
+  publicAddress: 'Publiek adres',
+  'auth.import.recoveryInstructions':
+    'Upload hier een JSON-wallet-herstelbestand om je accounts aan Neon toe te voegen. Deze optie is ook beschikbaar op de pagina Instellingen.',
+  importFile: 'Bestand importeren',
+  dashboardTokenBalancesPrice: 'PRIJS',
+  dashboardTokenBalancesHoldings: 'AANTAL',
+  settingsLanguageLabel: 'TAAL',
+  addToken: 'Token Toevoegen',
+  contactsPageLabel: 'Contacten Beheren',
+  newContact: 'Nieuw Contact',
+  deleteLabel: 'Verwijderen',
+
+  addToContacts: 'Toevoegen aan contacten',
+  contactName: 'Naam',
+  enterAContactName: 'Voer naam in',
+  enterAWalletAddress: 'Voer wallet adres in',
+  contactWalletAddress: 'Wallet adres',
+  editAContact: 'Pas een contact aan',
+  modifyDetails: 'Details aanpassen',
+  removeContact: 'Verwijder contact',
+  saveContactButtonText: 'Sla contact op',
+
+  editContactDisclaimer:
+    'Controleer of je het adres correct hebt ingevoerd om ervoor te zorgen dat je geen bezit verliest',
+  addAContact: 'Voeg een contact toe',
+  addContactDetails: 'Voeg contact-details toe',
+  confirmRemoveContact: 'Bevestig het verwijderen van contact',
+  modalActionConfirm: 'Akkoord',
+  modalActionCancel: 'Annuleren',
+  newsPageLabel: 'Niews',
+  networkSettingsLabel: 'Netwerk Instellingen',
+
+  walletManagerNoLocalInfo:
+    'Het lijkt erop dat je geen wallet-informatie lokaal hebt opgeslagen',
+
+  walletManagerRemoveWallet: 'Wallet verwijderen',
+
+  selectAssets: 'Select Assets',
+  priorityTransfer: 'Priority Transfer',
+
+  editRecipients: 'Edit Recipients',
+  confirmAndSend: 'Confirm & Send',
+  fee: 'Fee:',
+  sendMoreAssets: 'Send More Assets',
```

Comment:
Missing translation

Reply:
Missed that one. I took the liberty to change `fee` to `vergoeding` all over.